src/mobu/handlers/external.py

```diff
@@ -232,3 +253,33 @@ async def get_summary(
     context: Annotated[RequestContext, Depends(context_dependency)],
 ) -> list[FlockSummary]:
     return context.manager.summarize_flocks()
+
+
+@external_router.post(
+    "/github/webhook",
+    summary="GitHub webhooks",
+    description="This endpoint receives webhook events from GitHub.",
+    status_code=202,
+)
+async def post_github_webhook(
+    context: Annotated[RequestContext, Depends(anonymous_context_dependency)],
+) -> Response:
+    """Process GitHub webhook events.
+
+    This should be exposed via a Gafaelfawr anonymous ingress.
+    """
+    webhook_secret = config.github_webhook_secret
+    body = await context.request.body()
+    event = Event.from_http(
+        context.request.headers, body, secret=webhook_secret
+    )
+
+    # Bind the X-GitHub-Delivery header to the logger context; this
+    # identifies the webhook request in GitHub's API and UI for
+    # diagnostics
+    logger = context.logger.bind(github_delivery=event.delivery_id)
+
+    logger.debug("Received GitHub webhook", payload=event.data)
+    # Give GitHub some time to reach internal consistency.
+    await asyncio.sleep(1)
```

Comment:
I wish GitHub would have its act together before it sends the webhook. Maybe we should make this a configuration option just to make it a bit less of a magical constant? Or at least make the 1 a constant in `mobu.constants`?